content/docs/conditional-rendering.md

```diff
@@ -1,243 +1,245 @@
+Si bien declarar una variable y usar una sentencia `if` es una buena forma de renderizar condicionalmente un componente, a veces podrías querer usar una sintaxis más corta. Hay algunas formas de hacer condiciones $INLINE en JSX, explicadas a continuación.
 
-### Inline If with Logical && Operator
+### If $INLINE con operador lógico &&
 
-You may [embed any expressions in JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) by wrapping them in curly braces. This includes the JavaScript logical `&&` operator. It can be handy for conditionally including an element:
+Puedes [embeber cualquier expresión en JSX](/docs/introducing-jsx.html#embedding-expressions-in-jsx) envolviéndola en llaves. Esto incluye al operador lógico `&&` de JavaScript. Puede ser ùtil para incluir condicionalmente un elemento:
 
 ```js{6-10}
-function Mailbox(props) {
-  const unreadMessages = props.unreadMessages;
+function Buzon(props) {
+  const mensajesNoLeidos = props.mensajesNoLeidos;
   return (
     <div>
-      <h1>Hello!</h1>
-      {unreadMessages.length > 0 &&
+      <h1>¡Hola!</h1>
+      {mensajesNoLeidos.length > 0 &&
         <h2>
-          You have {unreadMessages.length} unread messages.
+          Tienes {mensajesNoLeidos.length} mensajes sin leer.
         </h2>
       }
     </div>
   );
 }
 
-const messages = ['React', 'Re: React', 'Re:Re: React'];
+const mensajes = ['React', 'Re: React', 'Re:Re: React'];
 ReactDOM.render(
-  <Mailbox unreadMessages={messages} />,
-  document.getElementById('root')
+  <Buzon mensajesNoLeidos={mensajes} />,
+  document.getElementById('raiz')
 );
 ```
 
-[**Try it on CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
+[**Pruébalo en CodePen**](https://codepen.io/gaearon/pen/ozJddz?editors=0010)
 
-It works because in JavaScript, `true && expression` always evaluates to `expression`, and `false && expression` always evaluates to `false`.
+Funciona porque en JavaScript, `true && expresión` siempre evalúa a `expresión`, y `false && expresión` siempre evalúa a `false`.
 
-Therefore, if the condition is `true`, the element right after `&&` will appear in the output. If it is `false`, React will ignore and skip it.
+Por eso, si la condición es `true`, el elemento just después de `&&` aparecerá en el resultado. Si es `false`, React lo ignorará.
 
-### Inline If-Else with Conditional Operator
+### $INLINE If-Else con operador condicional
 
-Another method for conditionally rendering elements inline is to use the JavaScript conditional operator [`condition ? true : false`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Conditional_Operator).
+Otro método para el renderizado condicional $INLINE de elementos es usar el operador condicional [`condición ? true : false`](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Conditional_Operator) de JavaScript.
```

Comment:
```suggestion
Otro método para el renderizado condicional $INLINE de elementos es usar el operador condicional [`condición ? true : false`](https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Operadores/Conditional_Operator) de JavaScript.
```

Reply:
Changed URL and fixing the suggestion to be consistent with the link.